src/server/scripts/Kalimdor/zone_darkshore.cpp

```diff
@@ -413,10 +413,117 @@ class npc_prospector_remtravel : public CreatureScript
     }
 };
 
+enum TwilightDisciplineThugSpells
+{
+    SPELL_RENEW = 6074,
+    SPELL_HEAL  = 2054,
+
+    SPELL_DISARM = 6713,
+};
+
+enum TwilightCreatures
+{
+    NPC_TWILIGHT_THUG = 2339,
+    NPC_TWILIGHT_DISCIPLINE = 2338,
+};
+
+enum TwilightDisciplineThugTalks
+{
+    TALK_ENGAGE = 0,
+    TALK_DEFEAT,
+};
+
+enum TwilightDisciplineThugEvents
+{
+    EVENT_HEAL = 1,
+    EVENT_RENEW,
+
+    EVENT_DISARM,
+};
+
+class npc_twilight_discipline_thug : public CreatureScript
+{
+public:
+    npc_twilight_discipline_thug() : CreatureScript("npc_twilight_discipline_thug") {}
+
+    struct npc_twilight_discipline_thug_AI : public ScriptedAI
+    {
+        npc_twilight_discipline_thug_AI(Creature* creature) : ScriptedAI(creature)
+        {
+        }
```

Comment:
keep these on the same line
```suggestion
        npc_twilight_discipline_thug_AI(Creature* creature) : ScriptedAI(creature) { }
```